mlflow/pipelines/steps/train.py

```diff
@@ -114,6 +119,8 @@ def _run(self, output_directory):
         tags = {
             MLFLOW_SOURCE_TYPE: SourceType.to_string(SourceType.PIPELINE),
             MLFLOW_PIPELINE_TEMPLATE_NAME: self.step_config["template_name"],
+            MLFLOW_PIPELINE_PROFILE_NAME: self.step_config["profile"],
```

Comment:
Can we also add a unit test to verify that these fields are correctly logged? Somewhere here: https://github.com/mlflow/mlflow/blob/master/tests/pipelines/test_train_step.py#L113

Reply:
I'll add the unit test in the next PR.

Reply:
I've extended existing unit test.